setup.py

```diff
@@ -12,5 +12,6 @@
     license='MIT (Expat)',
     url=madprops.__url__,
     packages=['madprops'],
-    install_requires=['django >= 1.4.11, < 1.7', 'djangorestframework > 3.0.0']
+    install_requires=[
+        'django >= 1.7', 'djangorestframework >= 3.3.0']
```

Comment:
I think for both requirements we should specify a range, since we cannot guarantee that our lisb will work for Django 2.0.0 (for example)